corehq/apps/users/tests/test_user_data.py

```diff
@@ -38,19 +38,8 @@ def test_user_data_accessor(self):
             'cruise': 'control',
             'this': 'road',
         })
-        # Normally you shouldn't use `user.user_data` directly - I'm demonstrating that it's updated
-        self.assertEqual(user.user_data, {
-            'cruise': 'control',
-            'this': 'road',
-        })
-        del user_data['cruise']
-        self.assertEqual(user.user_data, {
-            'this': 'road',
-        })
-        user_data.update({'this': 'field'})
-        self.assertEqual(user.user_data, {
-            'this': 'field',
-        })
+        # Normally you shouldn't use `user.user_data` directly - I'm demonstrating that it's not updated
+        self.assertEqual(user.user_data, {})
```

Comment:
nit: this test could be renamed to `test_couch_user_data_does_not_update` or something like that right? Partially asking to make sure I understand this.

Reply:
yeah that's correct - I think I named it this because it was the first basic test using the accessor.  I'll clean it up in the followup PR once the migration is fully rolled out